src/Microsoft.HttpRepl/Resources/Strings.resx

```diff
@@ -286,23 +274,23 @@ When +history option is specified, commands specified in the text file will be a
     <value>Unable to determine a base address</value>
   </data>
   <data name="ConnectCommand_Status_Swagger" xml:space="preserve">
```

Comment:
Change the name to "ConnectCommand_Status_OpenAPI" or something similar?

Reply:
I'm going to do a second pass at this later on to clean up the non-user-facing uses of swagger as appropriate.